File: SecondSemester/lab5/C#/Shape.cs
using System;

namespace lab5 {
    public class Shape {
        public double[,] coord;

        public Shape(double[,] coord) {
            this.coord = coord;
        }

        public double getLength(double x, double y, double a, double b) {
            return Math.Sqrt(Math.Pow((a - x), 2) + Math.Pow((b - y), 2));
        }
    }
}

File: SecondSemester/lab5/C++/Shape.cpp
#include "Shape.h"
#include <cmath>

Shape::Shape(double** coord, int size) {
    this->size = size;
    this->coord = coord;
}

double Shape::getLength(double x, double y, double a, double b) {
    return sqrt(pow((a - x), 2) + pow((b - y), 2));
}

Shape::~Shape() {
    for (int i = 0; i < this->size; i++)
        delete [] this->coord[i];
    delete [] this->coord;
}